### PR TITLE
feat: add PHP runtime version reporting (v2)

### PR DESCRIPTION
**Pull request by tomlongridge**
Augments reporting with the version of PHP in use when an event occurred in case of issues in a specific version of PHP. This is a change to v2 of the bugsnag-php libraries to enable Wordpress
support.

## Goal

Augments reporting with the version of PHP in use when an event occurred in case of issues in a specific version of the framework.

## Design

Added data to device.runtimeVersions.php as per other notifiers.

### Added

* Diagnostics.deviceData (array) - stores device data to be included in the report
* Diagnostics.mergeDeviceData - adds specified data to the device data object

### Changed

* Diagnostics.__construct - appends the runtime version information to the device data through the mergeDeviceData method

## Tests

* Added unit tests to ClientTest and DiagnosticTests
* Tested end-to-end through bugsnag/bugsnag-wordpress PR

## Review

For the submitter, initial self-review:

- [x] Commented on code changes inline explain the reasoning behind the approach
- [x] Reviewed the test cases added for completeness and possible points for discussion
- [x] A changelog entry was added for the goal of this pull request
- [x] Check the scope of the changeset - is everything in the diff required for the pull request?
- This pull request is ready for:
  - [ ] Initial review of the intended approach, not yet feature complete
  - [ ] Structural review of the classes, functions, and properties modified
  - [x] Final review

For the pull request reviewer(s), this changeset has been reviewed for:

- [x] Consistency across platforms for structures or concepts added or modified
- [x] Consistency between the changeset and the goal stated above
- [x] Internal consistency with the rest of the library - is there any overlap between existing interfaces and any which have been added?
- [x] Usage friction - is the proposed change in usage cumbersome or complicated?
- [x] Performance and complexity - are there any cases of unexpected O(n^3) when iterating, recursing, flat mapping, etc?
- [x] Concurrency concerns - if components are accessed asynchronously, what issues will arise
- [x] Thoroughness of added tests and any missing edge cases
- [x] Idiomatic use of the language

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+## TBD
+
+### Enhancements
+
+* Add PHP version string to report and session payloads (device.runtimeVersions)
```
Comment (mikeewhite) [inline CHANGELOG.md]: Does this branch include session tracking support?
